cmd/docker/builder.go

```diff
@@ -0,0 +1,123 @@
+package main
+
+import (
+	"bytes"
+	"fmt"
+	"os"
+	"strconv"
+
+	pluginmanager "github.com/docker/cli/cli-plugins/manager"
+	"github.com/docker/cli/cli/command"
+	"github.com/pkg/errors"
+	"github.com/spf13/cobra"
+)
+
+const (
+	builderDefaultPlugin     = "buildx"
+	builderDefaultInstallMsg = `To install buildx, see
```

Comment:
Do we want to make this one of the [`/go/` links](https://github.com/docker/docker.github.io/blob/7238e69b20872d20c9211e0d1dfd634b21db6dbb/go/about.md) so that we can update the documents over time?

Reply:
Done. Go link preview: https://docs-stage.docker.com/go/buildx